### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #220: Pick realistic comparator input to check our code with
## What feature would you like to have?

I'd like to have a standard comparator query-URL that we will then use whenever we're comparing a new piece of code to some old piece of code. This way it's easy to see what works in the design, and what doesn't work. Because we always use the same string, its easier to see what changes between versions.

The best thing to have would be a query-string that has realistic output, and uses as much of the different components as possible (codeblocks, headings, changelog text, etc...).

## What problem will be solved by having the feature?

Makes it easier for me to see what I need to change in the code to refine the design. 😄 

Have the link in a central place, so I don't have to go hunting in our issues for a suitable query string.

## Do you want to help making the feature?

I can't because I don't have a good idea of what the realistic testing input should be.

We should put this URL somewhere in a development document. 😄

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Create a basic `CONTRIBUTING.md` file with instructions on how to get started

## Context

We should have a document that explains how to get started with helping us.
This is beneficial for us too, as we can put things we need to remember while developing in here. 😉 

Fixes #220.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+1. Create a feature branch
+1. Start the development server with `yarn start`
+1. Make improvements
+1. Put chunks of work in a commit (the Husky program will run some checks)
+1. Run `yarn smoketest` to confirm you're not breaking anything critical
+1. Create pull request
+
+## Query string to check comparator output
+
+Use the following query string to check the comparator output: INSERT PROPER LINK HERE
```
Comment (HonkingGoose) [inline CONTRIBUTING.md:73]: @Belco90 Can you suggest a realistic query string we should use here?

Comment (Belco90) [reply CONTRIBUTING.md:73]: I tend to use this comparison for checking everything is working as expected: `?repo=testing-library%2Fdom-testing-library&from=v6.16.0&to=v8.1.0`

It includes major, minor, and patch semver, code of blocks, diffs, commits and PRs references, different text emphasis... I think this is a good candidate for what you are looking for.

**Review by Belco90 — APPROVED** [review]
Wow, fantastic addition. Thank you very much for adding these guidelines!

**Comment by HonkingGoose** [comment]
It needs more work, but for now this is good enough to start with. 😄

**Pull request merged**